spec/defines/openssl_export_pem_cert_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'spec_helper'
+
+describe 'openssl::export::pem_cert' do
+  let(:title) { '/etc/ssl/certs/foo.pem' }
+  let(:facts) do
+    {
+      path: '/usr/bin:/bin:/usr/sbin:/sbin',
+    }
+  end
+
+  context 'when enable and no pfx or der cert' do
+    let(:params) do
+      {
+        ensure:  :present,
+      }
+    end
+
+    it 'fails' do
+      expect {
+        is_expected.to contain_exec('Export /etc/ssl/certs/foo.pfx to /etc/ssl/certs/foo.pem')
+      }.to raise_error(Puppet::Error)
```

Comment:
Maybe check the error message content, since several failures are possible:


```suggestion
      }.to raise_error(Puppet::Error, /either/)
```